start.py

```diff
@@ -228,6 +228,14 @@ def get_constants(metadata):
     return constants
 
 
+def set_jvm_locale(m2ee_section, java_version):
+    javaopts = m2ee_section["javaopts"]
+
+    # enable locale for java8 or later
+    if not java_version.startswith("7"):
```

Comment:
The comment says "enable locale for java8 or later", but the code says "enable locale for java versions that are not 7".

Are we sure that the only Java versions are 7 and 8+?

Reply:
yep, we do not support java older than 7 in buildpack